src/main/flight/pid.c

```diff
@@ -833,7 +876,7 @@ static float getSterm(int axis, const pidProfile_t *pidProfile)
 #endif
 }
 
-NOINLINE static void calculateSpaValues(const pidProfile_t *pidProfile)
+static FAST_CODE_NOINLINE void calculateSpaValues(const pidProfile_t *pidProfile)
```

Comment:
IIRC `NOINLINE` was used to prevent speed-optimizing function when called from optimized/FAST code. This change does exactly the opposite ...

Reply:
I get confused by it, I have to say.
I've reverted that change.